pkg/replica/replica.go

```diff
@@ -1265,7 +1265,8 @@ func (r *Replica) UnmapAt(length uint32, offset int64) (n int, err error) {
 		// For list `unmappableDisks`, the first entry is the volume head,
 		// the second one is the parent of the first entry...
 		unmappableDisks := []string{r.diskPath(r.activeDiskData[len(r.activeDiskData)-1].Name)}
-		for idx := len(r.activeDiskData) - 2; idx > 0; idx-- {
+		indexOfVolumeHeadParent := len(r.activeDiskData) - 2
+		for idx := indexOfVolumeHeadParent; idx > int(backingFileIndex); idx-- {
```

Comment:
We should only skip the first disk (index:1) when the replica is based on a backing image only, right?

Reply:
Yes, only skip index 1 (backing image) and index 0 (empty struct) if a backing image exists.

Reply:
yeah, after updating this part, LGTM. cc @shuo-wu for this change/limitation.